### PR TITLE
Update Introduction.md

### PR DESCRIPTION
**Pull request by stanleycyang**
Updated the docs to use `classes` and `ReactDOM`. 

This allows the user to get the most updated approached to build React with react-router

### CODE REVIEW

**Comment by knowbody** [comment]
looks good, thanks. the only change I'd like you to do is to follow this style:

```js
import { render } from 'react-dom'
```

File: docs/Introduction.md [hunk 1]
```diff
@@ -7,27 +7,28 @@ To illustrate the problems React Router is going to solve for you, let's build a
 ### Without React Router
 
 ```js
-import React from 'react'
+import React, { Component } from 'react'
+import ReactDOM from 'react-dom'
 
 const About = React.createClass({/*...*/})
 const Inbox = React.createClass({/*...*/})
 const Home = React.createClass({/*...*/})
```
Comment (knowbody) [inline docs/Introduction.md]: what about line 12-14?

Comment (taion) [inline docs/Introduction.md]: The actual React Router code uses `React.createClass` everywhere, though, no? It might be better to be consistent with that.

**Comment by knowbody** [comment]
@taion yeah that's true, I use `classes` that's why it seemed correct for me. Difficult to convince @mjackson and @ryanflorence to classes.

But I think at least for now let's keep the `React.createClass` as we use `mixins` here and there.

**Comment by taion** [comment]
Hipsters :p

Probably for this PR, switch everything back to `React.createClass` for consistency then.

**Comment by knowbody** [comment]
:+1:

File: docs/Introduction.md (hunk 1 above)
Comment (stanleycyang) [inline docs/Introduction.md]: I can make that change.

**Comment by stanleycyang** [comment]
I think this change is going to come sooner or later, IMO. The `ReactDOM` part has already happened, and since everything is going towards ES6 and babel we maybe we can show both `ES5` and `ES6` examples but let people know that ES6 is what is going to be prevalent

**Comment by knowbody** [comment]
@stanleycyang we might do that later, if you want to add the react 0.14 bits and pieces please add the changes @taion asked for. Thanks :smile:

**Comment by taion** [comment]
Sooner or later, sure, but most likely not until there's a broadly accepted replacement for mixins - HoCs aren't sufficient yet for things that hook into lifecycle in some cases, unfortunately.

**Comment by stanleycyang** [comment]
Updated it to account for `import { render } from 'react-dom'` per @knowbody

**Comment by taion** [comment]
Thanks!

**Comment by knowbody** [comment]
thanks @stanleycyang

**Pull request merged**